### PR TITLE
minor additions and changes

### PR DESCRIPTION
**Pull request by chrivand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,26 @@
...
+Most of the root skeleton files are pulled from [Hashicorp's repository](https://github.com/hashicorp/terraform-provider-scaffolding) covering this topic.  However, in anecdotal experience, this repository can be overwhelming to those new to the process.  This repository is not meant as a replacement to the official repository, but meant as a barebones set of files and process to expedite the publishing process of new providers. 
+
+The main steps of the process of publishing a Terraform Provider via Cisco DevNet looks like this:
+
+1. Write, properly document, verify and test the Terraform Provider;
+2. Migrate the repository to CiscoDevNet (if not already there);
+3. Create GPG signing keys and securely share the public key with CiscoDevNet admin (most likely a Developer Advocate (DA));
+4. Add the appropriate GoReleaser files (can be found in this repository and [here](https://github.com/hashicorp/terraform-provider-scaffolding);
+5. Notify the CiscoDevNet admin you are working for, after which it can be published to the Terraform Registry;
+6. Share the Terraform Provider with the world! Have you created DevNet Learning Labs already? What about a Cisco Developer blog post?
```
Comment (annegentle) [inline README.md]: Maybe also add the info on how to start a LL or blog post? Who to contact?

File: README.md [hunk 2]
```diff
@@ -2,17 +2,26 @@
...
+6. Share the Terraform Provider with the world! Have you created DevNet Learning Labs already? What about a Cisco Developer blog post? 
 
 ## Included or Required Files
 
 This repository contains several files to assist with the building of the provider when a release is generated.
 
 ### GPG Key
 
-An ASCII-armored GPG key is required.  Raw keys are generally kept within each publishing BE, with the ASCII-armored export being added to the Terrform Registry under the CiscoDevNet organization.  You may need to work with your DA in order to add the key to the appropriate namespace.
+An ASCII-armored GPG key is required.  Raw keys are generally kept within each publishing Cisco Business Entirity (BE), with the ASCII-armored export being added to the Terrform Registry under the CiscoDevNet organization.  You may need to work with your DA in order to add the key to the appropriate namespace. If you don't have an existing GPG key, you can [generate a new GPG key](https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/generating-a-new-gpg-key) to use for signing commits and tags.
```
Comment (annegentle) [inline README.md]: Entity (instead of Entirity) I think

File: README.md (hunk 1 above)
Comment (chrivand) [reply README.md]: yes great idea, I guess the answer is to contact the DA?

File: README.md (hunk 2 above)
Comment (chrivand) [reply README.md]: yes sorry that was a typo from my side

**Review by qsnyder — APPROVED** [review]
LGTM!

**Pull request merged**